### PR TITLE
Prompt for release prep steps

### PR DESCRIPTION
**Pull request by laurenfrederick**
## Description

GitHub Copilot Prompt for the release preparation steps.

< !-- Provide context for the reviewer to understand the pull request. Include what changes were made and why. -->

## Issue number or link

#1441 

## Author checklist

- [ ] I have submitted a [Contributor License Agreement](https://github.com/CesiumGS/community/tree/main/CLAs) (only needed once).
- [ ] I have done a full self-review of my code.
- [ ] I have updated `CHANGES.md` with a short summary of my change (for user-facing changes).
- [ ] I have added or updated unit tests to ensure consistent code coverage as necessary.
- [ ] I have updated the documentation as necessary.

## Reviewer checklist

Thank you for taking the time to review this PR. By approving a PR you are taking as much responsibility for these changes as the author.

As you review, please go through the checklist below:

- [ ] Review and run all parts of the test plan on this branch and verify it matches expectations.
    - [ ] If the issue is a bug please make sure you can reproduce the bug in the main branch and then checkout this branch to make sure it actually solved the issue.
- [ ] Review the code and make sure you do not have any remaining questions or concerns. You should understand the code change and the chosen approach. If you are not confident or have doubts about the code, please do not hesitate to ask questions.
    - [ ] Code should follow the [C++ Style Guide](https://cesium.com/learn/cesium-native/ref-doc/style-guide.html)
- [ ] Review the unit tests and make sure there are no missing tests or edge cases.
- [ ] Review documentation changes and updates to `CHANGES.md` to make sure they accurately cover the work in this PR.
- [ ] Verify that the [Contributor License Agreement](https://github.com/CesiumGS/community/tree/main/CLAs) has been submitted, if needed.

### CODE REVIEW

File: .github/skills/prepare-release/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+name: prepare-release
+description: 'Prepare a Cesium Native release for a given version number. Use when asked to prepare, prep, or cut a release, bump the release version, or run release preparation for cesium-native. Verifies CI on the canonical CesiumGS/cesium-native main branch, reviews and expands CHANGES.md, and sets the version in package.json and CMakeLists.txt.'
```
Comment (j9liu) [inline .github/skills/prepare-release/SKILL.md]: **NIT**: redundant?

```suggestion
description: 'Prepare a Cesium Native release for a given version number. Use when asked to prepare or cut a release, bump the release version, or run release preparation for cesium-native. Verifies CI on the canonical CesiumGS/cesium-native main branch, reviews and expands CHANGES.md, and sets the version in package.json and CMakeLists.txt.'
```

File: doc/topics/release-process.md [hunk 2]
```diff
@@ -4,13 +4,16 @@ This is the process we follow when releasing a new version of Cesium Native.
 
 ## Prepare for Release
 
+Run the `/prepare-release` skill in [.github/skills/prepare-release/SKILL.md](../../.github/skills/prepare-release/SKILL.md) and provide the release version number. It performs the first four steps for you:
```
Comment (j9liu) [inline doc/topics/release-process.md]: The documentation build is failing because `../.github/skills/prepare-release/SKILL.md` is not included in the known paths. You can add it to the `docs/CMakeLists.txt` under `doxygen_add_docs`.

However, doing so makes the page look like so:

Image

I'm not sure how useful it is to view it that way -- I kinda prefer seeing it on the GitHub site itself, since it's GH specific, so maybe this could link to https://github.com/CesiumGS/cesium-native/tree/main/.github/skills/prepare-release/SKILL.md. But up to you what you think is most useful!

**Review by j9liu — COMMENTED** [review]
Thanks @laurenfrederick ! Just two small comments from me.

**Comment by j9liu** [comment]
Thanks @laurenfrederick !
